GNUmakefile

```diff
@@ -115,9 +115,14 @@ link-git-hooks: ## Install git hooks
 update-atlas-sdk: ## Update the atlas-sdk dependency
 	./scripts/update-sdk.sh
 
-# details on usage can be found in CONTRIBUTING.md under "Creating New Resource and Data Sources"
+# details on usage can be found in CONTRIBUTING.md under "Scaffolding initial Code and File Structure"
 .PHONY: scaffold
 scaffold:
 	@go run ./tools/scaffold/*.go $(name) $(type)
 	@echo "Reminder: configure the new $(type) in provider.go"
 
+# details on usage can be found in CONTRIBUTING.md under "Scaffolding Schema and Model Definitions"
+.PHONY: scaffold-schemas
+scaffold-schemas:
+	@tools/scaffold/scripts/schema-scaffold.sh $(name)
```

Comment:
should the `scaffold` command call the `scaffold-schemas`?

Reply:
It is not the initial idea, but can be looked into. The `scaffold-schemas` requires the generator_config.yml file to be defined with the mapping of the endpoints that will be used. [CLOUDP-221362: Adjust existing scaffolding command](https://jira.mongodb.org/browse/CLOUDP-221362) will include the generation of a template for the generator_config.yml file, but will require defining the endpoints and possibly additional configuration before running `scaffold-schemas`.